go/src/.gometalinter.json

```diff
@@ -7,5 +7,9 @@
     "gosimple",
     "misspell",
     "unconvert"
-  ]
+  ],
+  "Concurrency": 16,
```

Comment:
yeah ur right, actually i have written as comment for the specification like this because commenting is not allowed by json.